docs/core/plugins/fts_solr.md

```diff
@@ -154,10 +154,8 @@ mail_plugins {
   fts_solr = yes
 }
 
-plugin {
-  fts = solr
-  fts_solr = url=https://solr.example.org:8983/solr/ debug batch_size=1000
-}
+fts_driver = solr
+fts_solr_url = url=https://solr.example.org:8983/solr/ debug batch_size=1000
```

Comment:
This isn't right. And I think there aren't any more fts-solr changes after this pull request. These need to be split into separate settings:

fts_solr_url = https://...
fts_solr_batch_size = 1000

fts_solr_debug setting is gone

Reply:
I changed the commit in Part IV, where this was introduced.